template/nuxt-app/assets/definitions.styl

```diff
@@ -100,6 +111,14 @@ style-body()
 	fluid-font 'body', body-font-size, body-font-size-min
 	line-height body-line-height
 
+// Media query helpers
+on-desktop()
+	@media(min-width tablet)
+		{block}
+on-mobile()
+	@media(max-width tablet - 1px)
+		{block}
+
```

Comment:
I'd like people to standardize on these ones instead:

https://github.com/BKWLD/stylus-library/blob/eba97404cbbd5e8667b5db00f496525273ce32d1/lib/utils.styl#L119-L146

So there is only one way we're doing media query helpers.  Thus, you would use `tablet-up` and `tablet-down`

```suggestion
```